includes/class-sensei-question.php

```diff
@@ -411,8 +411,13 @@ public function get_question_grade( $question_id ) {
      * @param $question_type
      */
     public static function load_question_template( $question_type ){
+		$template_loaded = Sensei_Templates::get_template( 'single-quiz/question_type-' . $question_type . '.php' );
 
-        Sensei_Templates::get_template  ( 'single-quiz/question_type-' . $question_type . '.php' );
+		if ( ! $template_loaded ) {
```

Comment:
the difference is the file name. On line 414 I search for the file with underscores. If that exists, then the user is using the old template. If it does not, then I load the new template with dashes in the name since they are not overriding it.